jax/_src/scipy/signal.py

```diff
@@ -19,6 +19,8 @@
 
 import numpy as np
 import scipy.signal as osp_signal
+from scipy.signal._signaltools import _init_freq_conv_axes as osp_fft_init_freq_conv_axes
```

Comment:
I don't think we should rely on non-public scipy APIs (i.e. imports with a leading underscore). They may be changed at any time without any warning.

Reply:
Hum. I can C&P their code? or should I ignore?

Reply:
We can't copy code from scipy. I'd suggest looking at the function API (expected outputs given particular inputs) and write your own implementation in JAX that does the same thing.

Reply:
Ok. I will do it tomorrow (for me in Paris :) )

Reply:
There is also similar input processing code in `_convovle_nd` in this file, you could model your implementation off of that if it would be helpful.